frontend/src/components/Home/Home.js

```diff
@@ -119,118 +119,11 @@ export const Home = () => {
-        props.setFilter({ month: values.month, year: values.year });
-    }
-    return (
-        <div>
-            <Formik
-                enableReinitialize
-                initialValues={{ month: "January" }}
-                onSubmit={submit}>
-
-                <Form>
-                    <Field as="select" name="month">
-                        <option value={"January"}>January</option>
-                        <option value={"February"}>February</option>
-                        <option value={"March"}>March</option>
-                        <option value={"April"}>April</option>
-                        <option value={"May"}>May</option>
-                        <option value={"June"}>June</option>
-                        <option value={"July"}>July</option>
-                        <option value={"August"}>August</option>
-                        <option value={"September"}>September</option>
-                        <option value={"October"}>October</option>
-                        <option value={"November"}>November</option>
-                        <option value={"December"}>December</option>
-                    </Field>
-                    <Field type="number" name="year" />
-                    <button type="submit">
-                        Submit
-                    </button>
-                </Form>
-            </Formik>
-        </div>
-    );
-};
 
 
 
-const DeleteForm = (props) => {
-    let submit = (values) => {
-        const foundFinance = props.financesData.find(e => e.month == values.month && e.year == values.year);
-        props.deleteFinance(foundFinance._id);
-    }
-    return (
-        <div>
-            <Formik
-                enableReinitialize
-                initialValues={{ month: "January", year: 2024 }}
-                onSubmit={submit}>
-
-                <Form>
-                    <Field type="number" name="year" />
-                    <Field as="select" name="month">
-                        <option value={"january"}>january</option>
-                        <option value={"February"}>February</option>
-                        <option value={"March"}>March</option>
-                        <option value={"April"}>April</option>
-                        <option value={"May"}>May</option>
-                        <option value={"June"}>June</option>
-                        <option value={"July"}>July</option>
-                        <option value={"August"}>August</option>
-                        <option value={"September"}>September</option>
-                        <option value={"October"}>October</option>
-                        <option value={"November"}>November</option>
-                        <option value={"December"}>December</option>
-                    </Field>
-                    <button type="submit">
-                        Submit
-                    </button>
-                </Form>
-            </Formik>
-        </div>
-    );
-};
\ No newline at end of file
+
```

Comment:
Awesome, we need to rework most of it for the modern version, but it seems really good